templates/server.xml.erb

```diff
@@ -71,19 +75,22 @@
         />
 
 <% if @tomcat_native_ssl -%>
-        <Connector
-                    port="<%= @tomcat_https_port %>"
-                    protocol="<%= @tomcat_protocol_ssl_real %>"
-                    <%- if @tomcat_address -%>
-                    address="<%= @tomcat_address %>"
-                    <%- end -%>
-                    maxHttpHeaderSize="<%= @tomcat_max_http_header_size %>"
-                    SSLEnabled="true"
-                    maxThreads="<%= @tomcat_max_threads %>"
-                    minSpareThreads="<%= @tomcat_min_spare_threads %>"
-                    enableLookups="<%= @tomcat_enable_lookups %>"
-                    disableUploadTimeout="<%= @tomcat_disable_upload_timeout %>"
-                    acceptCount="<%= @tomcat_accept_count %>"
+        <Connector port="<%= @tomcat_https_port %>"
+                   <%- if Gem::Version.new(@version) >= Gem::Version.new('7.12.0') -%>
+                   relaxedPathChars="[]|" 
+                   relaxedQueryChars="[]|{}^&#x5c;&#x60;&quot;&lt;&gt;"
```

Comment:
same here?

Reply:
Hi @bastelfreak. Those are the entries specified in the Atlassian upgrade notes: 
https://confluence.atlassian.com/jirasoftware/jira-software-7-12-x-upgrade-notes-955173978.html
More precisely: "Add relaxedPathChars="[]|" relaxedQueryChars="[]|{}^&#x5c;&#x60;&quot;&lt;&gt;" to the connector properties in server.xml"